pkg/server/router/presentation.go

```diff
@@ -0,0 +1,140 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/tbd54566975/ssi-service/pkg/server/framework"
+	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
+	"github.com/tbd54566975/ssi-service/pkg/service/presentation"
+	"net/http"
+)
+
+type PresentationDefinitionRouter struct {
+	service *presentation.Service
+}
+
+func NewPresentationDefinitionRouter(s svcframework.Service) (*PresentationDefinitionRouter, error) {
+	if s == nil {
+		return nil, errors.New("service cannot be nil")
+	}
+	service, ok := s.(*presentation.Service)
+	if !ok {
+		return nil, fmt.Errorf("could not create presentation router with service type: %s", s.Type())
+	}
+	return &PresentationDefinitionRouter{service: service}, nil
+}
+
+type CreatePresentationDefinitionRequest struct {
+	PresentationDefinition exchange.PresentationDefinition `json:"presentation_definition" validate:"required"`
+}
+
+type CreatePresentationDefinitionResponse struct {
+	PresentationDefinition exchange.PresentationDefinition `json:"presentation_definition"`
```

Comment:
same here

Reply:
this response should be similar to https://github.com/TBD54566975/ssi-service/blob/main/pkg/server/router/manifest.go#L65

which does return the snake_case version because that is how the spec defines the property.

should this also return the JWT, or will you be handling that in another PR?

Reply:
wasn't sure if there was going to be a distinct endpoint for presentation requests. if there are - then disregard.